### PR TITLE
feat(xdsl.exchange): add rma call api

### PR DESCRIPTION
**Pull request by Steffy29**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#1179.0 ✘  
* 
  * build ✘  
  * test ✘

File: src/api/xdsl/rma/xdsl-rma.v6.service.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      isArray: true,
+      cache: queryCache,
+    },
+    get: {
+      method: 'GET',
+      cache,
+    },
+    getRMA: {
+      method: 'GET',
+      url: '/xdsl/:xdslId/rma/:id',
```
Comment (JeremyDec) [inline src/api/xdsl/rma/xdsl-rma.v6.service.js]: no cache used ?

Comment (Steffy29) [reply src/api/xdsl/rma/xdsl-rma.v6.service.js]: nope, need fresh info

File: src/api/xdsl/rma/xdsl-rma.v6.service.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  const interceptor = {
+    response(response) {
+      cache.remove(response.config.url);
+      queryCache.removeAll();
+      return response.resource;
+    },
+  };
+
+  const resource = $resource('/xdsl/:xdslId/rma', {
```
Comment (antleblanc) [inline src/api/xdsl/rma/xdsl-rma.v6.service.js]: Shouldn't be:

```suggestion
  const resource = $resource('/xdsl/:xdslId/rma/:id', {
```

instead?

File: src/api/xdsl/rma/xdsl-rma.v6.service.js (hunk 1 above)
Comment (antleblanc) [reply src/api/xdsl/rma/xdsl-rma.v6.service.js]: Why not prefer using the `get` (L22) without the cache entry?

File: src/api/xdsl/rma/xdsl-rma.v6.service.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    get: {
+      method: 'GET',
+      cache,
+    },
+    getRMA: {
+      method: 'GET',
+      url: '/xdsl/:xdslId/rma/:id',
+    },
+    updateRMA: {
```
Comment (antleblanc) [inline src/api/xdsl/rma/xdsl-rma.v6.service.js]: Why not naming it `update`?

File: src/api/xdsl/rma/xdsl-rma.v6.service.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      method: 'PUT',
+      url: '/xdsl/:xdslId/rma/:id',
+      interceptor,
+    },
+    deleteRMA: {
```
Comment (antleblanc) [inline src/api/xdsl/rma/xdsl-rma.v6.service.js]: Why not naming it `delete`?

File: src/api/xdsl/rma/xdsl-rma.v6.service.js (hunk 2 above)
Comment (Steffy29) [reply src/api/xdsl/rma/xdsl-rma.v6.service.js]: I need the list too

Comment (antleblanc) [reply src/api/xdsl/rma/xdsl-rma.v6.service.js]: That's the `query` action purpose.

**Pull request merged**